userspace/libsinsp/sinsp_filtercheck_fd.cpp

```diff
@@ -341,35 +341,7 @@ uint8_t* sinsp_filter_check_fd::extract_from_null_fd(sinsp_evt *evt, OUT uint32_
 	}
 	case TYPE_FILENAME:
 	{
-		if(evt->get_type() != PPME_SYSCALL_OPEN_E && evt->get_type() != PPME_SYSCALL_OPENAT_E &&
-			evt->get_type() != PPME_SYSCALL_OPENAT_2_E && evt->get_type() != PPME_SYSCALL_OPENAT2_E &&
-			evt->get_type() != PPME_SYSCALL_CREAT_E)
-		{
-			return NULL;
-		}
-
-		if(extract_fdname_from_creator(evt, len, sanitize_strings) == true)
```

Comment:
Here `extract_fdname_from_creator` will always return `false` if we are an enter event

```cpp
bool sinsp_filter_check_fd::extract_fdname_from_creator(sinsp_evt *evt, OUT uint32_t* len, bool sanitize_strings, bool fd_nameraw)
{
	const char* resolved_argstr;
	uint16_t etype = evt->get_type();

	if(PPME_IS_ENTER(etype))
	{
		return false;
	}
        ...
```

So putting together the 2 conditions this code is equivalent to `return NULL`